### PR TITLE
Commands: add command namespacing

### LINKED ISSUE #2: Parser: namespace bot commands
We should "namespace" the bot commands with a flag, such as !ac for aracred. This will avoid clashes with other bots that may not have namespaced commands. Example usage for the bot would be:

- `!ac signup foo github/foo`

### PR DESCRIPTION
**Pull request by crisog**
The bot now receives the message and checks for the correct namespace (which is **_!ac_**). If it's not correct, it will return a RequestHandlerError.

Closes #2

### CODE REVIEW

**Comment by crisog** [comment]
This had older commits. Will submit a correct PR with the lastest changes.

File: src/handlers/index.js [hunk 1]
```diff
@@ -2,10 +2,11 @@ const addAddressHandler = require('./addAddress')
 const helpHandler = require('./help')
 const signupHandler = require('./signup')
 
+// This is the handler for commands with !ac flag.
```
Comment (Evalir) [inline src/handlers/index.js:5]: This is not needed; let's remove this.

File: src/parser/detectHandler.js [hunk 2]
```diff
@@ -4,13 +4,20 @@ const handlers = require('../handlers/index')
 const noop = () => undefined
 
 module.exports = function detectHandler(message) {
-  const requestedHandler = message.split(' ').shift()
+  const requestedNamespace = message.split(' ')[0]
+  let requestedHandler = message.split(' ')[1]
```
Comment (Evalir) [inline src/parser/detectHandler.js:8]: Using array destructuring we can do this in a more readable way:
`const [requestedNamespace, requestedHandler] = message.split(' ')`

File: src/parser/detectHandler.js [hunk 3]
```diff
@@ -4,13 +4,20 @@ const handlers = require('../handlers/index')
...
   // If it's not a flag, we can safely ignore this command.
-  if (!requestedHandler.includes('!')) {
+  if (!requestedNamespace.includes('!')) {
     return noop()
   }
-
-  const receivedHandler = handlers.get(requestedHandler)
+  let receivedHandler
```
Comment (Evalir) [inline src/parser/detectHandler.js:13]: Let's avoid using let whenever possible.

**Pull request merged**